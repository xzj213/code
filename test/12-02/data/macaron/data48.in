59
3402 3402 1362 3402 1362 1362 1362 1362 1362 0 0 3402 0 1362 3402 1362 1362 1362 3402 1362 1362 3402 3402 3402 0 1362 0 0 0 0 1362 1362 1362 0 0 1362 1362 3402 0 3402 3402 0 1362 0 1362 1362 3402 3402 0 3402 1362 0 0 3402 3402 0 0 1362 3402
1362 1362 0 3402 0 0 3402 3402 3402 0 0 1362 3402 3402 1362 0 1362 3402 0 0 3402 1362 1362 3402 3402 0 3402 0 3402 0 1362 1362 3402 1362 1362 0 3402 1362 0 1362 3402 0 3402 3402 1362 0 1362 0 1362 1362 1362 3402 0 0 0 1362 1362 0 0
0 1362 1362 0 1362 0 1362 1362 1362 1362 0 3402 1362 3402 3402 1362 3402 1362 0 3402 3402 1362 3402 3402 3402 0 3402 1362 3402 1362 3402 3402 1362 1362 3402 1362 1362 3402 0 1362 3402 3402 0 3402 1362 1362 3402 3402 1362 0 0 0 0 0 0 0 0 0 3402
0 0 3402 1362 3402 0 0 3402 3402 1362 0 0 0 1362 0 3402 0 3402 1362 1362 0 0 1362 0 0 3402 1362 3402 0 0 0 3402 1362 1362 1362 0 1362 1362 3402 1362 1362 3402 1362 3402 0 1362 1362 3402 0 1362 0 3402 0 1362 1362 1362 3402 1362 1362
1362 3402 0 0 0 1362 0 3402 1362 0 1362 1362 3402 0 1362 1362 1362 3402 0 1362 1362 0 1362 0 3402 1362 0 0 1362 1362 3402 1362 0 3402 1362 0 3402 3402 3402 0 1362 0 3402 3402 3402 1362 0 0 3402 1362 0 1362 0 3402 0 0 1362 0 3402
3402 1362 1362 1362 1362 3402 3402 3402 0 1362 3402 0 0 1362 3402 1362 1362 3402 3402 1362 1362 3402 0 3402 0 1362 1362 3402 3402 3402 3402 1362 1362 1362 3402 0 0 3402 3402 0 3402 1362 1362 3402 0 0 1362 0 0 1362 1362 0 1362 3402 0 0 3402 3402 1362
1362 0 0 0 1362 3402 0 0 1362 3402 1362 3402 1362 3402 1362 1362 0 0 0 1362 1362 0 0 0 3402 1362 0 1362 1362 1362 3402 0 0 3402 3402 1362 1362 3402 3402 1362 1362 3402 1362 0 1362 1362 0 0 0 1362 1362 1362 1362 3402 1362 3402 1362 3402 3402
1362 0 0 3402 0 0 3402 3402 0 0 3402 3402 0 1362 1362 1362 1362 3402 0 3402 3402 0 3402 3402 1362 3402 3402 3402 3402 1362 0 3402 0 1362 1362 1362 1362 3402 3402 0 0 0 0 1362 0 0 0 0 0 3402 3402 1362 1362 1362 3402 1362 3402 1362 1362
0 3402 3402 0 3402 3402 3402 0 1362 1362 1362 1362 1362 1362 0 3402 0 0 1362 0 3402 3402 0 0 0 0 3402 3402 3402 1362 3402 1362 3402 3402 3402 1362 0 0 1362 1362 3402 3402 1362 0 3402 3402 0 1362 1362 3402 3402 3402 0 0 3402 1362 0 3402 3402
0 1362 1362 0 1362 3402 3402 1362 1362 1362 0 0 1362 3402 0 3402 0 3402 1362 3402 1362 1362 1362 0 0 0 1362 3402 0 0 3402 3402 1362 3402 0 3402 0 0 3402 1362 3402 0 3402 3402 3402 3402 1362 0 1362 0 0 3402 0 3402 1362 0 3402 3402 3402
3402 0 0 3402 3402 0 3402 1362 1362 1362 3402 1362 0 3402 3402 3402 3402 3402 3402 1362 0 0 1362 0 1362 3402 0 0 1362 3402 0 3402 3402 1362 0 1362 1362 3402 1362 1362 3402 0 1362 3402 1362 3402 3402 0 1362 0 3402 3402 3402 1362 1362 1362 3402 0 0
1362 3402 1362 1362 3402 1362 3402 1362 1362 0 1362 0 3402 3402 3402 0 3402 3402 3402 1362 0 0 3402 3402 1362 3402 3402 1362 0 3402 0 1362 0 1362 0 3402 3402 1362 3402 3402 3402 1362 3402 1362 3402 1362 3402 0 1362 1362 3402 0 1362 1362 1362 1362 0 1362 1981
0 0 0 0 1362 0 3402 0 1362 1362 3402 1362 1362 0 1362 3402 3402 3402 1362 0 0 1362 3402 1362 1362 0 3402 0 1362 0 1362 1362 0 3402 3402 3402 3402 3402 0 3402 1362 3402 1362 1362 1362 0 3402 3402 1362 0 1362 3402 0 1362 0 0 0 3402 3402
1362 3402 3402 3402 3402 1362 0 3402 0 1362 1362 0 1362 0 0 3402 0 0 3402 0 1362 3402 3402 0 1362 3402 1362 1362 1362 0 0 1362 3402 3402 0 1362 1362 1362 1362 1362 0 1362 3402 3402 1362 1362 0 3402 3402 3402 0 3402 3402 3402 0 3402 0 1362 1362
1362 0 3402 1362 0 1362 1362 1362 0 1362 3402 3402 0 3402 1362 3402 3402 1362 0 3402 0 1362 0 0 3402 1362 0 0 1362 0 1362 0 1362 0 0 3402 1362 0 3402 1362 0 3402 1362 3402 0 0 0 1362 3402 3402 0 3402 1362 3402 0 1362 1362 3402 1362
1362 0 3402 3402 3402 0 0 3402 1362 1362 0 3402 3402 3402 3402 1362 1362 0 3402 0 3402 3402 0 0 0 3402 0 1362 3402 1362 1362 0 0 1362 0 3402 3402 3402 3402 1362 1362 0 3402 1362 1362 0 0 3402 3402 0 1362 1362 3402 0 3402 1362 3402 1362 1362
3402 1362 0 1362 3402 3402 0 3402 1362 1362 0 0 3402 3402 1362 3402 3402 1362 3402 1362 1362 0 3402 1362 1362 3402 3402 1362 1362 1362 3402 1362 3402 0 3402 1362 0 0 3402 0 0 0 1362 0 0 3402 0 1362 1362 1362 3402 0 1362 3402 0 3402 1362 3402 0
0 1362 1362 1362 3402 1362 0 1362 1362 3402 0 3402 1362 1362 1362 1362 1362 1362 0 1362 0 1362 1362 0 1362 1362 1362 0 0 3402 1362 0 3402 1362 3402 1362 0 0 1362 3402 3402 3402 1362 0 3402 3402 3402 1362 0 0 1362 0 3402 3402 0 0 3402 1362 3402
1362 1362 3402 0 3402 0 0 0 3402 3402 1362 3402 3402 3402 0 1362 1362 3402 1362 1362 0 0 1362 1362 1362 1362 1362 1362 3402 1362 3402 1362 1362 3402 1362 3402 0 3402 0 1362 1362 0 0 1362 3402 3402 928 3402 3402 0 3402 3402 0 1362 1362 0 1362 0 0
3402 3402 0 1362 0 1362 1362 0 0 3402 0 3402 1362 1362 1362 1362 0 3402 0 0 3402 1362 0 0 0 1362 3402 1362 1362 3402 3402 0 0 1362 1362 1362 1362 0 0 3402 1362 3402 0 3402 0 0 1362 3402 0 3402 3402 3402 3402 1362 3402 3402 1362 3402 3402
3402 3402 3402 3402 1362 1362 3402 3402 3402 0 1362 3402 1362 3402 0 0 0 0 0 0 1362 0 1362 3402 3402 0 3402 0 1362 0 1362 3402 1362 0 3402 0 0 0 1362 0 0 3402 3402 0 3402 1362 1362 0 3402 1362 0 1362 3402 3402 1362 0 1362 1362 1362
3402 0 0 1362 1362 0 1362 3402 0 3402 1362 3402 0 3402 0 0 0 3402 1362 3402 0 3402 0 0 1362 1362 3402 0 0 3402 0 1362 3402 3402 0 0 1362 1362 1362 3402 3402 0 1362 1362 1362 0 0 3402 1362 1362 1362 0 3402 3402 3402 3402 0 3402 3402
3402 3402 0 3402 1362 3402 3402 3402 3402 0 1362 3402 0 3402 0 0 1362 1362 0 1362 3402 3402 1362 1362 1362 3402 1362 3402 1362 0 1362 1362 0 3402 3402 3402 1362 1362 3402 0 0 1362 0 1362 1362 0 1362 3402 1362 3402 3402 3402 1362 3402 1362 3402 1362 1362 0
0 3402 3402 0 1362 3402 0 3402 0 3402 3402 3402 1362 0 1362 3402 1362 0 0 0 3402 3402 3402 3402 0 1362 0 3402 0 3402 1362 0 0 3402 1362 3402 1362 3402 1362 3402 3402 3402 1362 0 0 0 3402 1362 1362 3402 3402 1362 3402 0 1362 0 1362 3402 1362
3402 0 3402 1362 1362 0 1362 3402 3402 1362 0 3402 0 1362 0 0 3402 1362 1362 0 0 3402 3402 1362 3402 1362 3402 1362 3402 3402 0 1362 3402 1362 1362 0 0 3402 1362 0 1362 0 1362 0 1362 3402 0 3402 3402 3402 3402 0 0 0 3402 0 3402 1362 0
1362 3402 1362 0 1362 3402 0 3402 3402 3402 1362 1362 1362 3402 1362 3402 3402 1362 1362 3402 3402 3402 3402 1362 3402 3402 0 1362 1362 1362 3402 3402 0 0 3402 0 1362 1362 1362 1362 1362 3402 1362 0 3402 0 1362 0 1362 0 1362 1362 0 1362 1362 3402 0 3402 0
1362 3402 3402 3402 1362 1362 3402 3402 3402 1362 0 0 1362 3402 1362 3402 1362 3402 1362 3402 1362 3402 1362 0 0 0 0 3402 3402 3402 1362 3402 3402 3402 3402 3402 0 0 0 0 1362 0 1362 3402 0 3402 3402 1362 3402 3402 1362 0 0 1362 3402 1362 1362 1362 1362
3402 1362 0 3402 0 0 3402 0 0 3402 1362 0 1362 1362 3402 0 3402 3402 1362 3402 1362 1362 3402 3402 1362 3402 1362 0 3402 1362 1362 0 0 3402 1362 3402 3402 1362 1362 0 3402 0 3402 3402 1362 0 0 1362 1362 3402 0 0 1362 0 0 0 3402 0 3402
3402 3402 0 1362 0 3402 1362 1362 1362 0 3402 3402 3402 0 3402 1362 1362 1362 3402 3402 3402 1362 0 0 0 0 3402 0 3402 1362 3402 0 1362 0 3402 3402 3402 1362 1362 3402 0 3402 0 0 1362 3402 0 1362 3402 1362 0 1362 0 3402 0 1362 0 0 0
3402 3402 0 3402 3402 1362 3402 3402 0 1362 3402 1362 1362 0 1362 0 3402 1362 3402 3402 1362 0 1362 3402 1362 3402 3402 1362 3402 0 0 1362 1362 0 3402 1362 0 3402 0 0 0 0 3402 0 0 3402 0 3402 1362 0 0 1362 0 0 0 3402 0 0 3402
1362 3402 3402 1362 1362 0 0 3402 0 3402 3402 1362 1362 1362 1362 0 1362 0 1362 0 3402 3402 3402 1362 3402 1362 3402 1362 3402 1362 1362 3402 0 0 0 1362 3402 0 0 1362 3402 0 1362 1362 3402 1362 0 1362 0 3402 0 3402 1362 3402 3402 1362 1362 1362 3402
1362 3402 1362 3402 3402 3402 0 1362 0 0 0 3402 1362 1362 3402 0 0 3402 1362 3402 3402 1362 1362 1362 1362 1362 1362 1362 1362 0 1362 0 1362 0 3402 3402 3402 0 0 1362 1362 3402 3402 0 0 1362 0 3402 3402 3402 0 0 3402 1362 1362 3402 3402 3402 0
1362 3402 0 3402 0 0 0 3402 3402 3402 0 1362 3402 3402 1362 3402 0 1362 3402 3402 0 0 3402 1362 0 1362 3402 0 1362 0 3402 3402 3402 1362 3402 0 3402 3402 3402 3402 0 3402 1362 0 1362 1362 0 1362 0 1362 1362 0 3402 3402 0 1362 1362 3402 1362
0 3402 3402 3402 0 1362 0 0 0 0 0 3402 1362 0 3402 3402 1362 1362 1362 3402 0 0 3402 0 3402 0 1362 3402 0 0 0 1362 1362 3402 1362 1362 1362 0 0 3402 1362 0 1362 3402 0 0 0 0 1362 1362 1362 3402 3402 1362 0 1362 3402 1362 3402
0 1362 1362 1362 0 0 1362 3402 1362 0 1362 0 3402 0 1362 3402 1362 0 0 0 1362 1362 3402 1362 1362 0 0 1362 1362 1362 1362 0 1362 3402 0 0 0 1362 0 3402 0 1362 1362 0 1362 3402 0 3402 0 0 3402 1362 0 1362 0 0 1362 1362 1362
0 0 1362 3402 1362 3402 1362 1362 0 0 1362 3402 1362 3402 0 0 0 0 3402 0 0 0 0 1362 1362 0 3402 3402 0 0 3402 3402 1362 1362 3402 0 3402 3402 3402 3402 0 3402 0 1362 1362 0 3402 1362 3402 0 3402 0 0 1362 3402 3402 1362 3402 0
3402 1362 3402 0 3402 3402 1362 0 0 0 3402 0 1362 3402 1362 3402 1362 3402 3402 3402 0 1362 1362 0 0 0 3402 3402 3402 0 0 3402 0 0 3402 0 1362 1362 0 1362 0 0 0 3402 1362 0 3402 0 0 3402 0 3402 3402 3402 0 1362 3402 3402 3402
0 0 3402 1362 0 3402 1362 0 0 0 0 3402 1362 1362 1362 3402 3402 1362 3402 1362 3402 0 0 3402 1362 1362 3402 0 1362 0 3402 3402 0 1362 1362 1362 3402 0 1362 1362 1362 0 3402 0 3402 1362 0 1362 1362 3402 3402 0 3402 0 0 3402 3402 1362 0
3402 1362 3402 1362 3402 3402 3402 0 1362 1362 0 1362 3402 1362 0 0 1362 1362 0 1362 3402 1362 1362 0 0 1362 1362 0 0 3402 1362 3402 3402 0 3402 1362 3402 3402 0 1362 3402 3402 0 0 3402 3402 0 0 3402 0 1362 3402 1362 3402 1362 1362 3402 3402 1362
1362 1362 1362 0 1362 1362 0 3402 0 1362 0 3402 1362 1362 1362 3402 3402 3402 3402 3402 0 1362 1362 1362 1362 1362 3402 1362 1362 0 0 1362 1362 3402 1362 0 3402 0 3402 3402 3402 3402 1362 1362 1362 1362 3402 1362 1362 0 1362 0 1362 3402 3402 3402 1362 3402 0
0 3402 1362 1362 3402 0 1362 3402 3402 1362 1362 3402 1362 0 3402 0 3402 3402 0 1362 0 1362 3402 3402 1362 3402 3402 1362 1362 3402 1362 3402 3402 1362 1362 1362 3402 1362 1362 0 1362 3402 3402 0 0 1362 3402 3402 0 0 0 1362 1362 3402 0 0 3402 1362 3402
1362 3402 1362 1362 1362 1362 3402 1362 0 1362 0 0 3402 3402 0 1362 0 0 3402 0 0 0 1362 3402 1362 3402 1362 1362 1362 1362 1362 3402 3402 1362 3402 0 0 1362 0 1362 3402 3402 0 1362 3402 0 1362 0 1362 3402 3402 3402 0 0 0 1362 1362 0 3402
0 3402 0 3402 0 3402 0 1362 0 0 1362 1362 1362 0 1362 3402 3402 3402 3402 0 1362 3402 3402 3402 3402 0 3402 0 0 1362 0 3402 0 3402 3402 3402 1362 0 3402 3402 1362 1362 0 1362 1362 1362 3402 0 1362 0 3402 1362 0 3402 1362 1362 3402 3402 3402
3402 3402 0 1362 1362 1362 1362 1362 3402 1362 0 3402 0 3402 3402 0 3402 1362 1362 0 1362 0 3402 0 0 0 3402 1362 1362 1362 1362 1362 3402 3402 0 0 3402 1362 0 3402 1362 1362 0 0 0 0 3402 3402 3402 3402 1362 1362 1362 0 3402 3402 3402 0 1362
3402 0 1362 3402 3402 0 0 1362 3402 3402 1362 3402 1362 0 1362 0 0 0 1362 3402 0 3402 1362 3402 3402 3402 1362 1362 1362 1362 1362 1362 1362 1362 0 1362 1362 3402 0 0 0 1362 0 0 3402 0 3402 3402 3402 1362 3402 0 1362 3402 0 3402 0 3402 3402
3402 1362 3402 0 0 3402 1362 3402 3402 1362 0 0 1362 0 0 3402 3402 0 0 0 1362 1362 1362 1362 1362 3402 1362 3402 0 3402 3402 3402 1362 3402 3402 3402 3402 3402 3402 3402 3402 3402 0 1362 3402 1362 3402 1362 0 1362 1362 3402 1362 1362 1362 1362 3402 3402 3402
1362 3402 0 0 1362 3402 1362 3402 0 1362 0 0 0 1362 1362 0 1362 1362 0 1362 0 3402 3402 3402 0 3402 1362 0 3402 0 0 1362 3402 3402 3402 3402 1362 3402 0 0 1362 3402 0 2424 0 0 1362 3402 3402 1362 0 0 0 1362 1362 0 3402 3402 3402
1362 3402 1362 3402 0 0 3402 3402 0 1362 3402 3402 0 1362 0 0 1362 3402 1362 1362 1362 3402 3402 3402 0 1362 3402 3402 3402 0 3402 0 0 0 3402 3402 0 1362 1362 0 0 0 1362 3402 1362 0 1362 0 1362 1362 0 0 1362 3402 3402 3402 0 0 1362
1362 0 0 3402 3402 0 0 1362 3402 3402 1362 0 1362 3402 3402 0 3402 3402 3402 3402 3402 3402 0 1362 1362 1362 3402 1362 1362 3402 1362 3402 1362 1362 1362 1362 0 3402 0 0 0 1362 3402 3402 0 0 3402 3402 1362 0 0 3402 1362 3402 0 3402 3402 3402 0
3402 3402 1362 3402 0 3402 0 0 1362 3402 0 3402 3402 3402 1362 0 3402 0 0 1362 0 1362 1362 1362 3402 0 0 0 0 0 1362 0 3402 3402 3402 1362 0 0 1362 0 0 1362 0 3402 3402 1362 1362 0 3402 0 1362 1362 1362 1362 3402 3402 1362 3402 1362
1362 1362 1362 0 3402 0 1362 0 3402 0 3402 0 1362 0 0 3402 0 0 3402 1362 0 3402 0 1362 1362 3402 3402 0 0 3402 1362 3402 1362 1362 3402 1362 0 0 0 3402 1362 0 0 3402 1362 0 1362 1362 3402 0 0 0 1362 0 0 1362 0 1362 1362
3402 3402 3402 0 3402 3402 1362 3402 1362 3402 3402 1362 3402 3402 0 3402 3402 0 3402 3402 1362 0 0 0 1362 1362 3402 1362 3402 0 0 1362 1362 3402 3402 3402 1362 0 0 1362 0 0 1362 1362 3402 1362 0 0 0 1362 3402 1362 0 0 1362 3402 0 3402 3402
3402 0 0 1362 3402 1362 1362 0 1362 1362 1362 1362 0 1362 1362 3402 3402 1362 3402 3402 1362 0 1362 3402 0 1362 3402 0 1362 1362 3402 0 3402 3402 3402 1362 0 1362 1362 3402 0 3402 3402 0 3402 3402 3402 1362 0 1362 1362 3402 3402 0 1362 1362 3402 0 3402
3402 0 0 1362 1362 3402 1362 1362 1362 0 3402 0 1362 0 3402 1362 0 3402 3402 1362 0 1362 3402 1362 0 3402 3402 3402 0 3402 3402 0 1362 0 3402 3402 3402 1362 3402 0 0 0 1362 0 3402 1362 0 3402 1362 3402 3402 0 0 0 3402 0 0 3402 3402
3402 3402 3402 0 3402 3402 3402 3402 0 1362 3402 1362 3402 1362 1362 1362 1362 1362 1362 3402 1362 3402 1362 0 1362 0 3402 3402 1362 3402 0 3402 1362 1362 3402 0 1362 3402 1362 3402 0 0 3402 0 0 0 1362 3402 0 1362 3402 1362 0 0 3402 1362 3402 1362 3402
0 3402 0 3402 1362 0 3402 0 3402 0 0 3402 0 0 3402 0 1362 1362 1362 0 3402 1362 3402 0 3402 1362 1362 3402 1362 1362 1362 1362 1362 1362 3402 1362 0 1362 1362 1362 0 0 3402 3402 3402 0 0 0 0 0 1362 3402 0 3402 0 3402 0 3402 3402
1362 0 0 3402 0 3402 3402 3402 3402 3402 3402 0 0 1362 0 1362 3402 0 1362 0 0 0 3402 0 3402 0 0 0 1362 0 1362 3402 0 0 0 0 3402 0 3402 1362 3402 1362 1362 3402 3402 3402 0 1362 0 0 1362 1362 1362 3402 1362 1362 1362 0 0
0 0 1362 1362 0 1362 0 3402 3402 1362 1362 0 3402 0 3402 0 0 1362 1362 1362 0 3402 0 3402 3402 1362 1362 1362 0 3402 0 0 3402 1362 0 3402 1362 0 3402 1362 3402 1362 0 3402 0 3402 1362 0 0 0 1362 1362 1362 1362 1362 3402 0 1362 3402
3402 3402 1362 0 1362 3402 1362 3402 3402 1362 3402 1362 1362 3402 1362 3402 0 3402 1362 0 0 1362 0 0 3402 1362 3402 0 3402 1362 1362 0 3402 0 3402 0 0 1362 0 1362 3402 1362 3402 3402 0 3402 1362 3402 0 0 1362 1362 0 3402 1362 0 3402 0 0
