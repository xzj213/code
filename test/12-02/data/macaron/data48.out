0 47 335 352 704 751 911 928 1053 1074 1362 1405 1757 1778 1938 1981 2072 2103 2391 2424 2776 2807 2967 3000 3077 3114 3402 3429 3781 3818 3978 4005 
